### ISSUE TITLE
Not Compatible With Steam Deck

### ISSUE THREAD

**Issue by NicholasHallman** [dianariyanto/virtual-display-linux#32]
This software causes a permanent black screen on a Steam Deck after rebooting.

I'm suggesting adding another message on the readme warning to never run this on a steam deck since there's no way to solve this other than to re-install the OS. It does say "At your own risk" but making it clear that it's not compatible with steam deck is maybe a good idea.

**Comment by AnyCubical** [comment]
![image](https://user-images.githubusercontent.com/56519373/223973850-07c99831-5022-440d-a9b7-bf16b07d315b.png)
A bit of reading would have helped

**Comment by NicholasHallman** [comment]
Absolutely, the device compatibility section does outline that only intel devices work. However, it's below the screen shot and the use at your own risk warning so not immediately visible when you load into the repo. My suggestion is to replace that warning with a summary of the device compatibility.

> ### ⚠Use at your own risk. Only compatible with integrated Intel cards ⚠

Or a `Does this device have an intel integrated graphics card (Y/N): ` prompt during installation.

I appreciate your reply and pointing out the existing documentation. Thanks.

**Comment by exxocism** [comment]
I SHOULD HAVE SEEN THIS BEFORE I BRICK MY DECK

**Comment by exxocism** [comment]
If you have already messed up with your deck, go to https://www.reddit.com/r/SteamDeck/comments/17f7v8g/easy_factory_reset_option_that_valve_doesnt_want/ and follow the procedure 5. It's the only way to recover the deck by itself.
